### ISSUE TITLE
Action when persisted config can't be read

### ISSUE THREAD

**Issue by jsign** [prologic/bitcask#96]
Just want to discuss the following part of `bitcask.go`:
```
	cfg, err = config.Decode(path)
	if err != nil {
		cfg = newDefaultConfig()
	}
```
In case `config.json` is corrupted or somehow is an invalid JSON because somebody wanted to edit it by hand; is ok to silently ignore the root cause of the `err` and assume a new default configuration?

**Comment by prologic** [comment]
No its not! This was a mistake  and fixed in #95 Which I can merge now if you like and keep working on that branch and put up new PRs as I cover more error cases?

**Comment by jsign** [comment]
Oh ok, don't hurry. Just knowing that will be change and eventually merged its ok!
